python/neuroglancer/chunks.py

```diff
@@ -44,6 +44,5 @@ def encode_jpeg(subvol):
     img.save(f, "JPEG")
     return f.getvalue()
 
-
 def encode_raw(subvol):
-    return subvol.tostring('C')
\ No newline at end of file
+    return subvol.tostring('F')
```

Comment:
I've actually been torn over this. While working on superdownsample, I noticed it was easier to perform the transformations in the code i fed to encode_raw, so I kept it as 'C'. What was your experience?